### PR TITLE
demo feedback impl

### PR DESCRIPTION
**Pull request by bazzani**
- Use substring in RemoveUnwantedCharFromDirVar

### CODE REVIEW

File: GitUpdateScript.sh [hunk 1]
```diff
@@ -48,7 +45,7 @@ function MoveToCorrectBranch(){
 	then
 	  echo "######################################################"
 	  echo "# Could NOT find $branch_I_want or $branch_I_want_dev branches              #"
-	  echo "# Please check that you are on the corred branch     #" 
+	  echo "# Please check that you are on the corred branch     #"
```
Comment (DavidT12) [inline GitUpdateScript.sh:42]: typo

File: GitUpdateScript.sh [hunk 2]
```diff
@@ -9,19 +9,10 @@ clear
...
-	full_dir=$home_dir$file_dir  #merge home dir with the dir the git repository is in
-	###remove git file name searched for, char by char###
-	full_dir=${full_dir%?}  #remove right char
-	full_dir=${full_dir%?}  #remove right char
-	full_dir=${full_dir%?}  #remove right char
-	full_dir=${full_dir%?}  #remove right char
-}
+
 function GoToGitRepositoryDirectory(){
 	cd  #go to home dir
```
Comment (bazzani) [inline GitUpdateScript.sh]: you could probably remove this command

Comment (DavidT12) [reply GitUpdateScript.sh]: done

File: GitUpdateScript.sh (hunk 1 above)
Comment (DavidT12) [reply GitUpdateScript.sh:42]: todo in next task

**Review by DavidT12 — APPROVED** [review]
ready to merge

**Pull request merged**